modules/installation-gcp-permissions.adoc

```diff
@@ -32,6 +32,13 @@ account requires the following permissions. If you deploy your cluster into an e
 .Required roles for creating network resources during installation
 * DNS Administrator
 
+.Required roles for installing into a shared VPC
+* Compute Instance Admin
```

Comment:
It should be `Compute Instance Admin (v1)` instead. 

FYI the credential requests for GCP, with 4.12.0-0.nightly-2022-10-15-094115, where both `roles/compute.instanceAdmin` and `roles/compute.instanceAdmin.v1` are mentioned, and the former is a subset of the latter, so we can mention the latter only, thanks! 
```
$ oc adm -a pull_secret.json release extract registry.ci.openshift.org/ocp/release:4.12.0-0.nightly-2022-10-15-094115 --cloud gcp --credentials-requests --to "cco-credrequests"
Extracted release payload from digest sha256:55d8660794fbf33031e83e3b3489dc3718290ccbba38ab056d02ffe4a25274c4 created at 2022-10-15T09:43:33Z
$ 
$ ls cco-credrequests/
0000_26_cloud-controller-manager-operator_16_credentialsrequest-gcp.yaml
0000_30_cluster-api_00_credentials-request.yaml
0000_30_machine-api-operator_00_credentials-request.yaml
0000_50_cloud-credential-operator_05-gcp-ro-credentialsrequest.yaml
0000_50_cluster-image-registry-operator_01-registry-credentials-request-gcs.yaml
0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml
0000_50_cluster-network-operator_02-cncc-credentials.yaml
0000_50_cluster-storage-operator_03_credentials_request_gcp.yaml
$ 
$ grep roles cco-credrequests/*.yaml -n
cco-credrequests/0000_26_cloud-controller-manager-operator_16_credentialsrequest-gcp.yaml:15:    - roles/compute.instanceAdmin
cco-credrequests/0000_26_cloud-controller-manager-operator_16_credentialsrequest-gcp.yaml:16:    - roles/compute.loadBalancerAdmin
cco-credrequests/0000_26_cloud-controller-manager-operator_16_credentialsrequest-gcp.yaml:17:    - roles/iam.serviceAccountUser
cco-credrequests/0000_30_cluster-api_00_credentials-request.yaml:16:    - roles/compute.instanceAdmin.v1
cco-credrequests/0000_30_cluster-api_00_credentials-request.yaml:17:    - roles/iam.serviceAccountUser
cco-credrequests/0000_30_cluster-api_00_credentials-request.yaml:18:    - roles/compute.loadBalancerAdmin
cco-credrequests/0000_30_machine-api-operator_00_credentials-request.yaml:17:    - roles/compute.admin
cco-credrequests/0000_30_machine-api-operator_00_credentials-request.yaml:18:    - roles/iam.serviceAccountUser
cco-credrequests/0000_50_cloud-credential-operator_05-gcp-ro-credentialsrequest.yaml:15:    - roles/iam.securityReviewer
cco-credrequests/0000_50_cloud-credential-operator_05-gcp-ro-credentialsrequest.yaml:16:    - roles/iam.roleViewer
cco-credrequests/0000_50_cluster-image-registry-operator_01-registry-credentials-request-gcs.yaml:18:    - roles/storage.admin
cco-credrequests/0000_50_cluster-image-registry-operator_01-registry-credentials-request-gcs.yaml:19:    - roles/iam.serviceAccountUser
cco-credrequests/0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml:18:    - roles/dns.admin
cco-credrequests/0000_50_cluster-network-operator_02-cncc-credentials.yaml:16:    - roles/compute.admin
cco-credrequests/0000_50_cluster-storage-operator_03_credentials_request_gcp.yaml:16:    - roles/compute.instanceAdmin
cco-credrequests/0000_50_cluster-storage-operator_03_credentials_request_gcp.yaml:17:    - roles/compute.storageAdmin
cco-credrequests/0000_50_cluster-storage-operator_03_credentials_request_gcp.yaml:18:    - roles/iam.serviceAccountUser
$